backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -236,15 +236,21 @@ private static void UpdateHostIfChanged(string host, ref Robot robot, ref List<s
         private static void UpdateCurrentAssetIfChanged(string newCurrentAsset, ref Robot robot,
             ref List<string> updatedFields)
         {
+
             if (newCurrentAsset.Equals(robot.CurrentAsset, StringComparison.Ordinal))
             {
                 return;
             }
 
             updatedFields.Add($"\nCurrentAsset ({robot.CurrentAsset} -> {newCurrentAsset})\n");
             robot.CurrentAsset = newCurrentAsset;
-        }
 
+            if (robot.CurrentAssetDeck != null)
+            {
+                return;
+            }
+
+        }
```

Comment:
this part doesn't do anything

Reply:
Nope its not - just kept it incase we add a few lines of code related to currentassetdeck below it which we might need to skip if we have CurrentAssetDeck as null

Reply:
It better to add it when we need it instead of adding code that we might need some day

Reply:
Done :)